pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -205,7 +227,13 @@ func (b *HybridBotanist) DeployETCD() error {
 				return err
 			}
 		}
-		if err := b.K8sSeedClient.DeleteService(b.Shoot.SeedNamespace, fmt.Sprintf("etcd-%s", role)); err != nil && !apierrors.IsNotFound(err) {
+		service := &corev1.Service{
```

Comment:
Can you remove this whole `DeleteService` code? It's no longer needed, it was needed a few months ago during a migration.